app/src/main/java/com/example/zuccstagram_signup/PostMaker.java

```diff
@@ -130,4 +145,25 @@ public void onFailure(@NonNull Exception e) {
                     }
                 });
     }
+    // code to upload image to cloud
+    private void uploadCloudImage() {
+        StorageReference postsRef = storageReference.child("images/posts.jpg");

```

Comment:
"images/posts.jpg" will be altered in future to better organize where photos are stored in storage